### PR TITLE
Remove the postgresql dump restore step when upgrading from 3.0

### PR DESCRIPTION
**Pull request by rhafer**
There is no postgresql version upgrade needed currently (so we can just
continue to work with the old database files). And the previous code was very
SLE11 specific. (e.g. the check for postgresql 9.1, broke the restore).

### CODE REVIEW

**Comment by rhafer** [comment]
This should go in together with https://github.com/crowbar/crowbar-core/pull/592

**Comment by jsuchome** [comment]
Should we remove `db_dump_location` from the schemas? Or rather leave it there for possible future reuse...

File: chef/cookbooks/postgresql/recipes/db_restore.rb [hunk 1]
```diff
@@ -1,47 +0,0 @@
-#
```
Comment (sayalilunkad) [inline chef/cookbooks/postgresql/recipes/db_restore.rb]: I think we should leave this recipe as it is right now till there is another way of doing this. For now it should be enough not to include it.

Comment (rhafer) [inline chef/cookbooks/postgresql/recipes/db_restore.rb]: What for? If we ever need it again it's easy to restore it from git. Leaving it lying around unused will just cause confusion IMO.

**Comment by rhafer** [comment]
> Should we remove db_dump_location from the schemas? Or rather leave it there for possible future reuse...

db_dump_location is an attribute of the crowbar barclamp. So if it needs to go away it should happen there.

**Comment by jsuchome** [comment]
+1

**Comment by rsalevsky** [comment]
+1

File: chef/cookbooks/postgresql/recipes/db_restore.rb (hunk 1 above)
Comment (sayalilunkad) [inline chef/cookbooks/postgresql/recipes/db_restore.rb]: ok, fair enough. +1

**Pull request merged**